cmd/oci-image-tool/create.go

```diff
@@ -40,9 +41,10 @@ func createHandle(context *cli.Context) error {
 	}
 
 	v := bundleCmd{
-		typ:  context.String("type"),
-		ref:  context.String("ref"),
-		root: context.String("rootfs"),
+		typ:      context.String("type"),
+		ref:      context.String("ref"),
+		root:     context.String("rootfs"),
+		platform: context.StringSlice("platform"),
```

Comment:
I made the following changes:
1. Added a filter function to filter the manifests
2. Added a more specific description of the platform flag, easy to use

PTAL, What other questions please put forward. Thanks.